src/services/leafletMapDefaults.js

```diff
@@ -101,69 +102,8 @@ angular.module('ui-leaflet').factory('leafletMapDefaults', function ($q, leaflet
         setDefaults: function (userDefaults, scopeId) {
             var newDefaults = _getDefaults();
 
-            if (isDefined(userDefaults)) {
```

Comment:
Angular extends don't work in complex objects, or object with functions, this could generate errors.